tutor/src/components/task-teacher-review/exercise.cjsx

```diff
@@ -16,7 +16,7 @@ TaskTeacherReviewQuestion = React.createClass
 
   onChangeAnswerAttempt: (answer) ->
     # TODO show cannot change answer message here
-    console.log('You cannot change an answer on a problem you\'ve reviewed.', 'TODO: show warning in ui.')
+    alert('You cannot change an answer on a problem you\'ve reviewed.')
 
```

Comment:
this make me nervous since an `alert` message is so ugly and we didn't display anything previously.  

@pandafulmanda do you have any idea how likely it is that this will be displayed?